src/components/views/rooms/EventTile.tsx

```diff
@@ -463,6 +458,7 @@ export class UnwrappedEventTile extends React.Component<IProps, IState> {
         client.removeListener(RoomEvent.Receipt, this.onRoomReceipt);
         this.isListeningForReceipts = false;
         this.props.mxEvent.removeListener(MatrixEventEvent.Decrypted, this.onDecrypted);
+        DecryptionFailureTracker.instance.removeVisibleEvent(this.props.mxEvent);
```

Comment:
I think this naming might be confusing, because rendered & visible are very different things. Visible implies in the visible section of the viewport. Can we get better naming or make reality match the naming?